components/clp-package-utils/clp_package_utils/scripts/native/archive_manager.py

```
@@ -285,11 +286,17 @@ def _find_archives(
-                    logger.warning(f"Archive {archive_id} in database not found on disk.")
+
+            if StorageType.FS == archives_output_config.storage.type:
+                archives_dir = archives_output_config.get_directory()
+                archive_output_dir: Path = (
+                    archives_dir / dataset if dataset is not None else archives_dir
+                )
+                for archive_id in archive_ids:
+                    logger.info(archive_id)
+                    archive_path = archive_output_dir / archive_id
+                    if not archive_path.is_dir():
+                        logger.warning(f"Archive {archive_id} in database not found on disk.")
```

Pylint (3.3.7)

[warning] Use lazy % formatting in logging functions

(W1203)